scripts/build_ios.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PROJECT_DIR=$(pwd)
+
+# Check for the device type argument
+if [[ "$1" == "simulator" ]]; then
+    DEVICE_TYPE="simulator"
+    ARCHITECTURE="aarch64-apple-ios-sim"
+elif [[ "$1" == "device" ]]; then
+    DEVICE_TYPE="device"
+    ARCHITECTURE="aarch64-apple-ios"
+else
+    echo -e "${RED}Error: Please specify either 'simulator' or 'device' as the first argument.${DEFAULT}"
+    exit 1
+fi
+
+# Check for the build mode argument
+if [[ "$2" == "debug" ]]; then
+    BUILD_MODE="debug"
+    LIB_DIR="debug"
+elif [[ "$2" == "release" ]]; then
+    BUILD_MODE="release"
+    LIB_DIR="release"
+else
+    echo -e "${RED}Error: Please specify either 'debug' or 'release' as the second argument.${DEFAULT}"
+    exit 1
+fi
+
+# build circom circuits in mopro-core
+
+cd ${PROJECT_DIR}/mopro-core/examples/circom/keccak256
+npm install
+./compile.sh
+cd ${PROJECT_DIR}/mopro-core
+cargo build --release
+
+# build ffi in mopro-ffi
+cd ${PROJECT_DIR}/mopro-ffi
+rustup target add x86_64-apple-ios aarch64-apple-ios aarch64-apple-ios-sim
+cargo install --bin uniffi-bindgen --path .
+make
+cargo build --release
+
+# build MoproKit pods
+cd ${PROJECT_DIR}/mopro-ios/MoproKit/Example
+pod install
+
+# update bindings
+cd ${PROJECT_DIR}
+./scripts/update_bindings.sh $1 $2
```

Comment:
Can we use BUILD_MODE and LIB_DIR as defined? More meaningful and less error prone